examples/storage/couchdb/README.md

```diff
@@ -0,0 +1,51 @@
+
+## Couchdb 2.0.0 on Kubernetes
+This document describes the steps to create a Couchdb 2.0.0 cluster on Kubernetes.
+
+### Prerequisites
+Access to a kubernetes cluster running v1.4 or later, kubectl installed and configured.  Minikube was used for this example. Note that the service type for the couchdb cluster is set to "NodePort" as the loadbalancer type is unsupported by minikube. 
```

Comment:
pls use Kube's standard for line wrapping at 80col and single space for sentences.